tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# tasks file for ansible-playbook-telegraf
+#
+
+- name: fetch telegraf installation package
+  get_url: url=http://get.influxdb.org/telegraf/telegraf_{{ telegraf_version }}_amd64.deb dest=/tmp/telegraf_{{ telegraf_version }}_amd64.deb mode=0440
+
+- name: install telegraf package
+  command: dpkg --skip-same-version -i /tmp/telegraf_{{ telegraf_version }}_amd64.deb
```

Comment:
It looks like the [apt module](http://docs.ansible.com/apt_module.html#options) takes a `deb` option which is probably safer than shelling out.